index.html

```diff
@@ -356,7 +356,7 @@ <h3>
             <dd>
               <p>
 Decentralized Identifiers (DIDs) are a new type of identifier for verifiable,
-decentralized digital identity, fully under the control of the DID
+decentralized digital identity, typically under the control of the DID
```

Comment:
I'm concerned that this `typically` modifies not only "under the control of the DID subject" but also "independent of any centralized registry"

Reply:
Maybe it could just be:
`decentralized digital identity, under the control of the DID`

Reply:
Another option is to just talk about what DIDs enable instead of couching this with "typically". That puts the focus on what new capabilities they bring and what they are for -- without saying that you *could* technically forgo these benefits.

`Decentralized Identifiers (DIDs) are a new type of identifier for verifiable, decentralized digital identity. These new identifiers are designed to enable DID subjects to prove control over them and to be implemented independent of any centralized registry, identity provider, or certificate authority.`

Reply:
The group agreed that we should use this text: "Decentralized Identifiers (DIDs) are a new type of identifier for verifiable, decentralized digital identity. These new identifiers are designed to enable the controller of a DID to prove control over them and to be implemented independently of any centralized registry, identity provider, or certificate authority."

Reply:
Forgot to apply this in the PR, but applied it in 09e7c0a560da6e5da922ba70aa8b7eb438cb83ee.